app/templates/content/buyers-guide-g-cloud.html

```diff
@@ -0,0 +1,247 @@
+<thead><tr>
+<th>Criteria number</th>
+<th>Direct award criteria</th>
+</tr></thead>
+<tbody>
+<tr>
+<td>1</td>
+<td>Whole life cost: cost effectiveness; price and running costs</td>
+</tr>
+<tr>
+<td>2</td>
+<td>Technical merit and functional fit: coverage, network capacity and performance as specified in relevant service levels</td>
+</tr>
+<tr>
+<td>3</td>
+<td>After-sales service management: help desk, account management function and assurance of supply of a range of services</td>
+</tr>
+<tr>
+<td>4</td>
+<td>Non-functional characteristics</td>
+</tr>
+</tbody>
+</table>
+
+<h3 id="">
+<span class="number">1.5 </span>Select your service, award and call-off contract
+</h3>
+
+<h4 id="single-supplier">Single supplier</h4>
+<p>
+If only one supplier meets your requirements, you can award the contract to them without doing anything else.
+</p>
+
+<h4 id="multiple-suppliers">Multiple suppliers</h4>
+<p>
+There will usually be a number of services on your shortlist. If all these services meet your requirements and are very similar, you can select one on the basis of lowest cost alone.
+</p>
+
+<p>Often the cheapest option will be obvious, but you might need to:</p>
+
+<ul>
+<li>look at combinations of different suppliers       	 </li>
+<li>get specific information about volumes</li>
+</ul>
+<p>The final selection should be based on best fit rather than ruling out suppliers that don’t meet either your current contract or an ideal set of terms.</p>
+
+<p>If you need any clarification to help you make your final decisions, you can request this directly from the suppliers. </p>
+
+<h4>Call-off contract</h4>
+<p>
+In a call-off contract, the terms and conditions and prices are all agreed when you sign the framework agreement. A copy of the call-off contract must be signed by all interested parties (the buyer and supplier) before the service can be used.
+</p><p>
+The G-Cloud framework is unlike most other frameworks because it includes the suppliers’ terms and conditions. In the case of any contradiction between the supplier's service terms and conditions in the call-off contract and the framework terms and conditions, the framework terms will take precedence.
+</p>
+
+<h3 id="">
+<span class="number">1.6 </span>Complete the Crown Commercial Service (CCS) savings form
+</h3>
+<p>
+CCS works with departments and organisations across the public sector to improve service delivery quality and ensure every commercial relationship provides value for money. That means it has to record savings and monitor the ongoing performance of the G-Cloud framework(s).
+</p><p>
+You must fill in the CCS
+<a  rel="external" href='https://ccs.cabinetoffice.gov.uk/sites/default/files/contracts/G-Cloud%20Customer%20Savings%20Record_0.docx'>G-Cloud Customer Savings Record</a> form every time you enter into a call-off agreement. Email a copy of your G-Cloud Customer Savings Record form to <a href='mailto:gcloud-savings@ccs.gsi.gov.uk'>gcloud-savings@ccs.gsi.gov.uk</a>.
```

Comment:
Not our problem necessarily, but should someone tell CCS that this link is insecure?